### ISSUE TITLE
[Feature Request] Add a preview of the copied pictures

### ISSUE THREAD

**Issue by t3m8ch** [Tudmotu/gnome-shell-extension-clipboard-indicator#295]
In Copyq it is possible to preview the copied picture. This is really useful when working with screenshots, when you make several screenshots and you can send them one by one without having to save them as a file.

**Comment by RafeeDaBoy** [comment]
I don't think this can show images that you copy. If it does, did you change something? I checked the settings and couldn't find anything

**Comment by Tudmotu** [comment]
Images are not supported yet, though there is a PR #286 .
Hopefully I will get to testing it soon.
